spec.html

```diff
@@ -31045,12 +31045,12 @@ <h1>Number.NEGATIVE_INFINITY</h1>
 
       <emu-clause id="sec-number.parsefloat">
         <h1>Number.parseFloat ( _string_ )</h1>
-        <p>The value of the `Number.parseFloat` data property is the same built-in function object that is the initial value of the *"parseFloat"* property of the global object defined in <emu-xref href="#sec-parsefloat-string"></emu-xref>.</p>
+        <p>The initial value of the `Number.parseFloat` property is the same function object as the initial value of the *"parseFloat"* property of the global object defined in <emu-xref href="#sec-parsefloat-string"></emu-xref>.</p>
```

Comment:
Would an alternative way to phrase it be to reference `%parseFloat%`?

Reply:
Yup, so it could be as short as:
```The initial value of the `Number.parseFloat` property is %parseFloat%.```
and then the `%parseFloat%` would auto-link to [19.2.4 parseFloat](https://tc39.es/ecma262/#sec-parsefloat-string).

Similarly with `Number.parseInt`.

For `Array.prototype [ @@iterator ]`, we could say
```The initial value of the @@iterator property is %Array.prototype.values%.```
but `%Array.prototype.values%` wouldn't auto-link anywhere, so we could add
```defined in <emu-xref href="#sec-array.prototype.values"></emu-xref>```

And then similarly with all the remaining ones.

Reply:
Seems more straightforward to me :-)

Reply:
@ljharb: Done